### ISSUE TITLE
md-select - multiple selections - does not work in iOS VoiceOver

### ISSUE THREAD

**Issue by benjiallen** [angular/material#11037]
**Actual Behavior**:
 -  `What is the issue? *`
    * In iOS VoiceOver, once an `md-select` with multiple selections is expanded, there is no way to collapse the list nor navigate to elements beyond the last option in the expanded option list
    * "Swiping right" past the last item in the `md-select` takes you outside of the viewport and into the browser chrome i.e., focus is moved to the back button
 -  `What is the expected behavior?`
    * There should be a method for closing the select menu once the user has made a selection
    * After you reach the end of the list "swiping right" should move focus to the top of the list

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:` https://material.angularjs.org/latest/demo/select#option-groups
 -  `Details:` Using iOS VoiceOver. Open the select menu for "topping", make a selection and then try to close the form control or move to the next form control on the page using the "swipe right" gesture.

**AngularJS Versions**: *
 -  `AngularJS Version:` 1.5.5
 -  `AngularJS Material Version:` 1.1.5

**Additional Information**:
 -  `Browser Type: *` Safari and VoiceOver
 -  `Browser Version: *` Latest that comes with iOS
 -  `OS: *` iOS 11.2.1
 -  `Stack Traces:` N/A

**Related issues**:

#10967

**Comment by Splaktar** [comment]
Thank you for adding this information to https://github.com/angular/material/issues/10967. I'm going to close this one as a duplicate and track this issue there.

**Comment by benjiallen** [comment]
Since this issue calls out issues with making multiple selections, should I add a comment to #10967 to capture this issue? It's not documented there.

**Comment by Splaktar** [comment]
@benjiallen yes, please do.

**Comment by benjiallen** [comment]
@Splaktar Done.
